plenum/config.py

```diff
@@ -354,3 +354,5 @@
 # - "quorum" - remove replicas only with quorum of BackupInstanceFaulty
 REPLICAS_REMOVING_WITH_DEGRADATION = "local"
 REPLICAS_REMOVING_WITH_PRIMARY_DISCONNECTED = "local"
+
+PRE_VC_STRATEGY = PreVCStrategies.VC_START_MSG_STRATEGY
```

Comment:
Maybe we should disable the strategy for now (until it's validated by load testing)?